lib/parallel_tests/test/runner.rb

```diff
@@ -99,7 +134,24 @@ def execute_command(cmd, process_number, num_processes, options)
 
           print_command(cmd, env) if report_process_command?(options) && !options[:serialize_stdout]
 
-          execute_command_and_capture_output(env, cmd, options)
+          result = []
+          process_in_batches(cmd, 8191, options[:files].first).map do |subcmd|
```

Comment:
I don't get the `options[:files].first)` part, why is the first file the suffix for all other files ?
(tested with start_with? in process_in_batches)

Reply:
... I'd split the commands at the first file

Reply:
the common parameters can appear after the files as well .. hence i was doing a starts_with ..

Reply:
thx, updated in my draft, plz see if that looks alright